examples/basics/basics.ipynb

```diff
@@ -36,13 +36,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
+   "id": "indie-bracket",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install labelbox"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
```

Comment:
What does `execution_count` do? Should it just be `1` for all cells?

Is there a way to restore that to a default amount.

Also I would probably use `nblint` just as a nice formatting tool on all notebooks.

https://github.com/alexandercbooth/nblint

Reply:
It is the number of times the cell has been called. I don't want to clear the notebooks because the outputs are useful if a user just wants to look at the expected behavior. I will look iinto nblint though.

Reply:
Ok, I added this code formatter to the repo. https://github.com/Labelbox/labelbox-python/blob/examples/examples/format_notebooks.py. It runs yapf with the same style used in the rest of the sdk. I also changed the execution count to be the order that the cells should be run. I don't want to totally clear the notebook because I think the notebooks will be referenced like documentation.